Index: .idea/sonarlint/issuestore/index.pb
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\n_\n/src/br/com/screenmatch/principal/Principal.java\u0012,b\\3\\b384b15eb06f47013b23a5639f6cb3a7b3ada50d
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/sonarlint/issuestore/index.pb b/.idea/sonarlint/issuestore/index.pb
--- a/.idea/sonarlint/issuestore/index.pb	(revision 8efd6fae4431851d6f8bddb92799cf83c3c7f948)
+++ b/.idea/sonarlint/issuestore/index.pb	(date 1686579288838)
@@ -1,3 +1,3 @@
 
-_
-/src/br/com/screenmatch/principal/Principal.java,b\3\b384b15eb06f47013b23a5639f6cb3a7b3ada50d
\ No newline at end of file
+}
+M.idea/shelf/Uncommitted_changes_before_Update_at_07_06_23_17_06__Changes_.xml,f\c\fc89836f5af1416f307300d09e316c53d70656fb
\ No newline at end of file
